### DISCUSSION TITLE
Support Multiserver MYSQL

### DISCUSSION THREAD

**Discussion by uploadx10** [Zrips/CMI#7272] [Enhancement]
Visit https://github.com/Zrips/CMI/discussions/new to post new idea or suggestion
Posting new suggestion in ticket section could result in their removal.
This makes saving and loading data for the Multiserver not lost.

**Comment by mrfloris** [comment]
As per the feature request info when you click on the button, please post feature requests under the discussions tab.

**Comment by CloudeLecaw** [comment]
https://www.spigotmc.org/resources/mysql-player-data-bridge.8117/
